p2p/discover/table.go

```diff
@@ -339,10 +342,16 @@ func (tab *Table) doRevalidate(done chan<- struct{}) {
 
 	// Also fetch record if the node replied and returned a higher sequence number.
 	if last.Seq() < remoteSeq {
-		n, err := tab.net.RequestENR(unwrapNode(last))
-		if err != nil {
+		n, enr_err := tab.net.RequestENR(unwrapNode(last))
+		if enr_err != nil {
 			tab.log.Debug("ENR request failed", "id", last.ID(), "addr", last.addr(), "err", err)
```

Comment:
```suggestion
			tab.log.Debug("ENR request failed", "id", last.ID(), "addr", last.addr(), "err", enr_err)
```
Don't we need to log `enr_err`?